Public/Src/Tools/SymbolDaemon/VsoSymbolClient.cs

```diff
@@ -46,7 +47,7 @@ public sealed class VsoSymbolClient : ISymbolClient
 
         private VssCredentials GetCredentials() =>
             new VsoCredentialHelper(m => m_logger.Verbose(m))
-                .GetCredentials(m_config.Service, true, null);
+                .GetCredentials(m_config.Service, true, null, null, PromptBehavior.Never);

```

Comment:
Is PromptNever the desired behavior? I thought the fix Christian needed was to prompt

Reply:
His fix is in the CB repo. Here, I'm just leaving us with he same behavior as before.